chat/README.md

```diff
@@ -1,2 +1,6 @@
 # Chatroom
 This example demonstrates how to simply create a chat application with system managed state.
+
```

Comment:
I do not think this insertion should be here; the purpose of all of our demos is to motivate simple install with `nim deploy project`.

Reply:
If there is.a change to the build procedure, that should go in `web/build.sh` and `web/build.cmd`.  The user can read those to understand what the build consists of.